src/Aquifer.Common/Services/NotificationService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Aquifer.Common.Jobs;
+using Aquifer.Common.Jobs.Messages;
+
+namespace Aquifer.Common.Services;
+
+public interface INotificationService
+{
+    public Task SendProjectStartedNotificationAsync(int projectId, CancellationToken cancellationToken);
+    public Task SendResourceCommentCreatedNotificationAsync(int commentId, CancellationToken cancellationToken);
+}
+
+public sealed class NotificationService(IQueueClientFactory _queueClientFactory) : INotificationService
+{
+    public async Task SendProjectStartedNotificationAsync(int projectId, CancellationToken cancellationToken)
+    {
+        var message = new SendProjectStartedNotificationMessage(projectId);
+
+        var queueClient = await _queueClientFactory.GetQueueClientAsync(Queues.SendProjectStartedNotification, cancellationToken);
+        await queueClient.SendMessageAsync(message, cancellationToken: cancellationToken);
+    }
```

Comment:
Hopefully this simple and easy to use syntax demonstrates why I wrote all of the utility code in this PR for publishing messages.